### DISCUSSION TITLE
Eufy Wifi Door Lock

### DISCUSSION THREAD

**Discussion by omar10wahab** [fuatakgun/eufy_security#368] [Q&A] [answered]
Hey is there plans to implement the WiFi door lock? I saw an older thread from last year which made it seem like it worked out but if I look at the latest description, it says no wifi. Wondering what might've changed or the future possiblity of getting the WiFi lock implemented. Thanks in advance

**Comment by omar10wahab** [comment]
Let me know in the comment section below please

**Comment by fuatakgun** [comment, marked as answer]
As far as these devices are supported by eufy-security-ws system, you can use them in HA. Giving a try might be easier to find out the answer

**Reply by omar10wahab** [reply]
You're 100% right. After verifying a couple of the steps needed to make the addon work which I forgot about that you do during initial install. I finally realized I needed to allow access to the device on my second account and I think that's it. Something else went wrong that caused some other issues with my existing doorbell but after rechecking and updating to the latest version, everything showed up and is working. Really really appreciate the work you put into this!
